content/guides/tinacms/meta.json

```diff
@@ -0,0 +1,4 @@
+{
+  "title": "Tina CMS",
+  "weight": 100
+}
```

Comment:
I added this file, but I'm not entirely sure what it does.  It doesn't seem to be influencing the nav at all (that is based off the `toc_doc`).